packages/odata-service-writer/templates/extend/manifest.json

```diff
@@ -57,7 +57,7 @@
             <% } %>
             <% if (locals.localAnnotationsName && typeof localAnnotationsName !== 'undefined') { %>,
                 <% if (locals.localAnnotationsName && typeof localAnnotationsName !== 'undefined') { %>
-                    <% if (Array.isArray(localAnnotationsName)) { %>
+                    <% if (Array.isArray(localAnnotationsName) && localAnnotationsName.length > 0) { %>
```

Comment:
Any tests for these changes?

Reply:
There is https://github.com/SAP/open-ux-tools/blob/main/packages/odata-service-writer/test/unit/updates.test.ts#L153 for multiple local annotations, but I can add test for `localAnnotationsName.length === 0` case.